packages/hub/src/graphql/types/ModelExport.ts

```diff
@@ -7,11 +12,47 @@ export const ModelExport = builder.prismaNode("ModelExport", {
     variableName: t.exposeString("variableName"),
     variableType: t.exposeString("variableType"),
     docstring: t.exposeString("docstring"),
+    isCurrent: t.exposeBoolean("isCurrent"),
     title: t.exposeString("title", { nullable: true }),
+    owner: t.field({
+      type: Owner,
+      select: {
+        modelRevision: {
+          select: {
+            model: {
+              select: {
+                owner: {
+                  select: {
+                    user: true,
+                    group: true,
+                  },
+                },
+              },
+            },
+          },
+        },
+      },
+      resolve: async ({ modelRevision: { model } }) => {
```

Comment:
I got curious about your `There might be a way to do this using "select" or "resolve", but I couldn't get those to work.` comment in the previous version of this code, so I investigated and ended up pushing this fix: 5d09d478de6dc4fbada337e7a2c32b681da29ceb.

The root cause was the explicit "include" field in `queries/exports.ts`, which overwritten `...query` above.

(The way this works in Pothos is that it analyzes all GraphQL fields below the current field and aggregates all necessary `include/select` props to that `query` object, so that the final Prisma query selects only the fields requested in  GraphQL. So you can rely on `t.field({ select: ... }` only if everything above the current field definition used Pothos-provided `...query`).